### ISSUE TITLE
[Bug Report] Change event on v-text-field firing twice

### ISSUE THREAD

**Issue by leebickmtu** [vuetifyjs/vuetify#10477]
### Environment
**Vuetify Version:** 2.2.10
**Vue Version:** 2.6.11
**Browsers:** Chrome 80.0.3987.85
**OS:** Mac OS 10.14.6

### Steps to reproduce
Use v-text-field with change event listener.

### Expected Behavior
- Change events only trigger if the value changed.
- And only once per change
  - triggered by either Enter key or blur, but not both, whichever happened first

### Actual Behavior
Since issue #2982, which resulted in Enter key triggering a change event, I have a few concerns.

- The change event is firing every time Enter key is pressed, even if the value has not changed.
  - The change event fired on blur only triggers if the value did change
- The change event is being fired again on blur after already having been fired on Enter key press.

### Reproduction Link
<a href="https://codepen.io/leebickmtu/pen/gOpOMRw" target="_blank">https://codepen.io/leebickmtu/pen/gOpOMRw</a>

**Comment by KaelWD** [comment]
Related: #1682

**Comment by Suxsem** [comment]
I can confirm the same bug.
This is a huge problem for me because I'm querying a large database when the value changes, so every time the user change the value in the text field, press enter and then click on another element the same query is executed twice.
Can you please take a look?

**Comment by johnleider** [comment]
Duplicate of #5070

If you have any questions, please reach out to us in our [Discord community](https://community.vuetifyjs.com).
